drivers/display/display_sdl.c

```diff
@@ -428,6 +428,43 @@ static int sdl_display_read(const struct device *dev, const uint16_t x,
 	return 0;
 }
 
+static int sdl_display_clear(const struct device *dev)
+{
+	const struct sdl_display_config *config = dev->config;
+	struct sdl_display_data *disp_data = dev->data;
+	size_t size = 0;
```

Comment:
drop the initialization from this one, that way the compiler fires a warning if there's a code path that uses it without assigning a values